sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/models/TransferCallResponse.java

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.callingserver.models;
+
+import com.azure.communication.callingserver.implementation.accesshelpers.TransferCallResponseConstructorProxy;
+import com.azure.communication.callingserver.implementation.models.TransferCallResponseInternal;
+import com.azure.core.annotation.Immutable;
+
+import java.util.Objects;
+
+/** The TransferCallResponse model. */
+@Immutable
+public final class TransferCallResponse {
```

Comment:
...Response suffix to be updated to be ...Result (per review) for .NET. Does this apply here also?

Reply:
So far we don't see any related comments on this yet

Reply:
Fixed